### DISCUSSION TITLE
how to use personal icon and use lottie file

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#666] [Q&A]
how to use personal icon and lottie file in nicegui

**Comment by falkoschindler** [comment]
Please, be more specific when opening a discussion. What do you mean with "personal icon"?

And, please, search through existing issues and discussions before opening a new one. Lottie files have been discussed here: https://github.com/zauberzeug/nicegui/discussions/262.

Maybe our brand new Discord server is also a good place for quick Q&A like this: https://discord.gg/TEpFeAaF4f

**Comment by stdusr** [comment]
Perhaps he means a favicon? :)

**Comment by bobwatcherx** [comment]
Eva icons , like a non-standard icon from nicegui . other than material
icons

Pada tanggal Rab, 29 Mar 2023 23.50, standard_user ***@***.***>
menulis:

**Comment by rodja** [comment]
```py
from nicegui import ui, Client

@ui.page('/')
async def main(client: Client):
    ui.add_head_html('<script src="https://unpkg.com/eva-icons"></script>')
    ui.element('i').props('data-eva="github"')
    await client.connected()
    await ui.run_javascript('eva.replace();', respond=False)

ui.run()
```

**Reply by bobwatcherx** [reply]
That's good . Suggestions if you can add documentation. it will be useful for others thanks

**Reply by rodja** [reply]
Good suggestion. It's a bit more tricky to add this than I thought:

```py
def more() -> None:
    @text_demo('Eva icons', '''
        You can use [Eva icons](https://akveo.github.io/eva-icons/#/) in your app.
    ''')
    async def eva_icons():
        from nicegui import Client

        # @ui.page('/')
        # async def main(client: Client):
        #     ui.add_head_html('<script src="https://unpkg.com/eva-icons"></script>')
        #     ui.element('i').props('data-eva="github"')
        #     await client.connected()
        #     await ui.run_javascript('eva.replace();', respond=False)
        # END OF DEMO
        ui.add_head_html('<script src="https://unpkg.com/eva-icons"></script>')
        ui.element('i').props('data-eva="github"')
        await ui.run_javascript('eva.replace();', respond=False)
```

Maybe I need to brainstorm with @falkoschindler how we do this kind of demos...

**Comment by rodja** [comment]
I just added a lottie demo to [ui.icon](https://nicegui.io/documentation/icon#lottie_files) and [ui.image](https://nicegui.io/documentation/image#lottie_files). Also improved the [eva icon demo](https://nicegui.io/documentation/icon#eva_icons).
